e2etest/bgptests/bgp.go

```diff
@@ -662,6 +662,33 @@ var _ = ginkgo.Describe("BGP", func() {
 				return nil
 			}, 4*time.Minute, 1*time.Second).ShouldNot(HaveOccurred())
 
+			ginkgo.By("checking the sessions don't flap when changing the configuration")
+
+			previousNeighbors := map[string]frr.NeighborsMap{}
+			for _, c := range FRRContainers {
+				neighbors, err := frr.NeighborsInfoMap(c)
+				Expect(err).NotTo(HaveOccurred())
+				previousNeighbors[c.Name] = neighbors
+			}
+			ginkgo.By("creating another the service")
+			svc1, _ := testservice.CreateWithBackend(cs, testNamespace, "external-local-lb1", tweak)
+			testservice.Delete(cs, svc1)
```

Comment:
is the Delete really need to observe the bug?

Reply:
nah, I meant it to be the usual deferred deletion

Reply:
will let is as is or use a defer?

Reply:
oops :-)
Fixed